### PR TITLE
User image

### PR DESCRIPTION
**Pull request by zachholcomb**
# PR Description
Please include a summary of the change:
Adds user image column to User table
Adds user image updating functionality

Commit Emojis:
:tada: User Image

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Adds new updating functionality for the users

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: code/api/src/migrations/6-user-image.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('users', 'image', {
+      type: Sequelize.STRING,
+      defaultValue: "https://images.unsplash.com/photo-1537151608828-ea2b11777ee8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=939&q=80"
```
Comment (rachael-t) [inline code/api/src/migrations/6-user-image.js:7]: Thanks for adding the defaultValue image!

File: code/api/src/seeders/1-user.js [hunk 2]
```diff
@@ -20,6 +21,7 @@ module.exports = {
         email: 'user@crate.com',
         password: bcrypt.hashSync('123456', config.saltRounds),
         role: params.user.roles.user,
+        description: 'Fashionista gurl',
```
Comment (rachael-t) [inline code/api/src/seeders/1-user.js:24]: Thanks for adding in these descriptions for the existing users!

**Review by rachael-t — APPROVED** [review]
Looks great and seems clear as to how we need to make the mutation request for the edits. Great job getting testing done as well!

File: code/api/src/seeders/1-user.js (hunk 2 above)
Comment (Karlfunhouse) [reply code/api/src/seeders/1-user.js:24]: this makes sense.  We will just need to update the photo and shipping address then right?

File: code/api/src/migrations/6-user-image.js (hunk 1 above)
Comment (Karlfunhouse) [reply code/api/src/migrations/6-user-image.js:7]: Will we change the defaultValue property when we change the image or will we create a new image property?

File: code/api/src/seeders/1-user.js (hunk 2 above)
Comment (zachholcomb) [reply code/api/src/seeders/1-user.js:24]: So this user update endpoint will eventually be able to update all of the user fields: so email, image description, etc.
now that the endpoint is set up we just have to update the arguments going into the function, which we'll do later today

Comment (zachholcomb) [reply code/api/src/seeders/1-user.js:24]: right now it is set to just update the user image

**Pull request merged**